src/translations/en.json

```diff
@@ -4365,7 +4367,9 @@
               "add_icon": "Add icon",
               "add_category": "Add category",
               "add_labels": "Add labels",
-              "add_area": "Add area"
+              "add_area": "Add area",
+              "supports_markdown": "[%key:ui::components::todo::item::supports_markdown%]",
+              "markdown": "[%key:ui::components::todo::item::markdown%]"
```

Comment:
Can we define these in a single common place somewhere? They're just duplicated once for now but we could use them for other inputs in the future